### PR TITLE
Updates tests using factory

### PR DESCRIPTION
**Pull request by maxcell**

### CODE REVIEW

File: test/controllers/user_controller_test.exs [hunk 1]
```diff
@@ -26,11 +30,14 @@ defmodule Streamr.UserControllerTest do
     end
 
     test "when a user exists with the email", %{conn: conn} do
-      conn = post conn, "api/v1/users/new", %{"user" => @valid_user_attrs}
+      valid_user = params_for(:user)
+
+
```
Comment (stevenpetryk) [inline test/controllers/user_controller_test.exs]: Could you remove this whitespace?

File: test/support/factory.ex [hunk 2]
```diff
@@ -0,0 +1,13 @@
+defmodule Streamr.Factory do
+  # with Ecto
+  use ExMachina.Ecto, repo: Streamr.Repo
+
+
```
Comment (stevenpetryk) [inline test/support/factory.ex]: And thees space

**Review by stevenpetryk — DISMISSED** [review]
alt text

File: test/controllers/user_controller_test.exs (hunk 1 above)
Comment (GregKWhite) [reply test/controllers/user_controller_test.exs]: Pls

File: test/support/factory.ex (hunk 2 above)
Comment (GregKWhite) [reply test/support/factory.ex]: pls

**Review by GregKWhite — APPROVED** [review]
LGTM :+1:

**Pull request merged**